### PR TITLE
PER-9148: Back-end: write "trigger directive" endpoint

### PR DESCRIPTION
**Pull request by liam-lloyd**
This is built atop the PER-9100 PR, so if that isn't merged yet comparing to the `per-9100` branch will show what's actually unique to this PR.

### CODE REVIEW

**Review by cecilia-donnelly — COMMENTED** [review]
It's taking me an unexpectedly long time to actually test this locally, so sharing these comments in the meantime!

File: src/legacy_client.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import fetch from "node-fetch";
```
[the remark this answers is not in the document]
Comment (liam-lloyd) [reply src/legacy_client.ts:1]: I'm a bit concerned that the built in `fetch` is _too_ newly available for us (see the issue on `permanent-sdk`).

**Comment by liam-lloyd** [comment]
To generate the JWT you need to hit this endpoint locally
1. Register your FusionAuth admin account with the `admin-local` application
2. Hit curl https://permanent-dev.fusionauth.io/api/login -H "Authorization: <FUSIONAUTH API KEY>" -H "X-FusionAuth-TenantId: <FUSIONAUTH TENANT ID>" -H "Content-Type: application/json" --data '{"applicationId": "<ADMIN LOCAL APPLICATION ID", "loginId": "<YOUR EMAIL>", "password": "<YOUR PASSWORD>"}' --verbose

File: src/legacy_client.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (cecilia-donnelly) [reply src/legacy_client.ts:1]: Perfect, we can have the discussion over there.

**Review by cecilia-donnelly — APPROVED** [review]
This looks good to me! Thanks a lot for working through it with me. Your comment on the PR about getting an admin token is very helpful - do you think there's a place for that inside the repo somewhere?

**Comment by liam-lloyd** [comment]
> Your comment on the PR about getting an admin token is very helpful - do you think there's a place for that inside the repo somewhere?

I was thinking it would go into Confluence as part of documentation on how to trigger directives through Retool.

**Pull request merged**